ui/client/src/components/agentic-ai/graphs/GraphDisplayHelpers.ts

```
@@ -0,0 +1,402 @@
+    id: string,
+    bounds: { x: string; y: string; w: string; h: string },
+    shadows: Record<string, string>[],
+  ) => {
+    if (defs.querySelector(`#${id}`)) return;
+    const f = document.createElementNS("http://www.w3.org/2000/svg", "filter");
+    f.setAttribute("id", id);
+    f.setAttribute("x", bounds.x);
+    f.setAttribute("y", bounds.y);
+    f.setAttribute("width", bounds.w);
+    f.setAttribute("height", bounds.h);
+    shadows.forEach((s) => f.appendChild(mkDropShadow(s)));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)